src/k8s-engine/k8s.ts

```diff
@@ -174,6 +181,18 @@ export interface KubernetesBackend extends events.EventEmitter {
    */
   setIntegration(name: string, state: boolean): Promise<string | undefined>;
 
+  /**
+   * If called after a backend operation fails, this returns a block of data that attempts
+   * to give more information about what command was being run when the error happened.
+   */
+  getFailureDetails(): Promise<FailureDetails>;
+
+  /**
+   * A description of the last backend command, usually displayed by the progress tracker,
+   * but available for the `FailureDetails` block.
+   */
+  lastCommandComment: string;
```

Comment:
Sorry, I don't see what's confusing about this.

Reply:
Of course my comment complaining about lack of clarity has a lack of clarity ;)

This property is only public so that people can set it (and people are not expected to read it); hence I think it might be clearer as `setLastCommandComment(string)`, to be clearer that (from the outside of this class) it isn't meant to be read.

Reply:
Implemented getters and setters for both commands and comments.